app/policies/project_policy.rb

```diff
@@ -14,7 +14,7 @@ def create?
   end
 
   def update?
-    user.id == project.user_id
+    user.id.in? project.user_ids
```

Comment:
Why do you use this method?

Reply:
This is from an old homework. Because we have several users who can participate in one project, so everyone should be able to edit the project.